dask/dataframe/tests/test_io.py

```diff
@@ -523,11 +512,10 @@ def test_Series_from_dask_array():
 def test_from_dask_array_compat_numpy_array():
     x = da.ones((3, 3, 3), chunks=2)
 
-    msg = r"from_array does not input more than 2D array, got array with shape \(3, 3, 3\)"
-    with tm.assertRaisesRegexp(ValueError, msg):
+    with pytest.raises(ValueError):
```

Comment:
I had to change how that function was implemented, and pulled the error checking into our code (which changed what they read). I didn't see the value in checking that the messages matched, but could change back if desired.